backend/src/main/java/ca/bc/gov/hlth/mohums/webclient/WebClientService.java

```diff
@@ -15,6 +19,18 @@ public WebClientService(WebClient kcAuthorizedWebClient) {
         this.kcAuthorizedWebClient = kcAuthorizedWebClient;
     }
 
+    public Mono<Object> getClients(List<String> authorizedClients) {
```

Comment:
This is just my opinion, but my suggestion is to keep `WebServiceClient` "dumb". All it should do is call Keycloak and return the results. If some controller's want to do something with the results, that's they're responsibility. So `ClientController` would be the brains of the operation. That's just my idea, and if you like it as it is that's fine by me.

Reply:
Looping in @tschia on this as I followed how he initially set things up for getGroups().

Reply:
Sorry I could have been a little clearer. This suggestion was that instead of passing in `authorizedClients` as a method parameter, just have `getClients` return all the clients. Then in the `ClientController` filter the clients there. Then all of the "logic" is happening in one place. Again, this is just a suggestion and a personal preference. The code is fine the way it is.

Reply:
Gotcha. I moved the filtering logic into the controller in this commit -> https://github.com/bcgov/moh-keycloak-user-management/pull/27/commits/1bb8016d9c05218ccdb1742de2e289115bf0f6bc

Reply:
I tend to agree with David, it might be good to try and keep the logic in one place (i.e. the controllers). 
In theory you could then just have a method that's `WebClientService.get(String path)` and have a generic get for all endpoints. (haven't thought out how this might work with request parameters)